.github/ISSUE_TEMPLATE/4. epic breakdown.yml

```
@@ -0,0 +1,98 @@
+name: Epic breakdown
+description: Template for breaking down PDP epics for engineering
+labels: ["kind/epic"]
+type: epic
+body:
+  - type: dropdown
+    id: Component
+    attributes:
+      label: Component
+      description: For which component your improvement idea is for?
```

🔧 This sounds like originating from a Feature Request template but not really fitting the context of a breakdown template? As it likely can be multiple, should this rather be a multi-select?
```suggestion
      description: What component is affected by this epic?
```

---

Good point. Changed

---

I will also remove the "not sure" option, since this should not be the case with an Epic.